recipes/nas/all/conanfile.py

```diff
@@ -17,7 +17,7 @@ class NasRecipe(ConanFile):
     topics = ("audio", "sound")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://www.radscan.com/nas.html"
-    license = "Unlicense"
+    license = "DocumentRef-wave.h:LicenseRef-MIT-advertising"
```

Comment:
What does it mean?

Reply:
We're reevaluating our understanding of the spdx specification. The docs right now point contributors to do something like "LicenseRef-FILENAME", but upon further inspection, I realized that the filename should be i the DocumentRef section, and that LicenseRef should be an id for the license. Probably this one one is actually incorrect too, and should be something like `DocumentRef-wave.h:LicenseRef-MIT-advertising-nas-variant` or something like that

Link here https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/

Reply:
Good question, I honestly don't know yet - licensing is always confusing!

Reply:
I asked in opensource stackexchange about it. We could use this pattern as well, but the answer was using something simpler like `LicenseRef-nas`, then add some fields in the license file. 

https://opensource.stackexchange.com/a/14309/31147

Anyway, I'm fine with this current approach too, as it's documented.